cg/cli/store.py

```diff
@@ -14,8 +14,15 @@
 
 
 @click.group()
-def store():
+@click.pass_context
+def store(context):
```

Comment:
Personal opinion, but if no command is invoked, it can print help and return, which could add to user friendliness!
It is also possible to cover this block with tests (providing context with fake config dict, and checking exit code to see if all api initiated correctly given the config variables). No need to do it though, if it doesn't seem helpful

Reply:
Next time! I need to get this into production now